src/components/fields/location-field/location-modal/location-modal.tsx

```diff
@@ -211,6 +215,10 @@ const LocationModal = ({
 		handleCloseLocationModal();
 	};
 
+	const handleHidePermissionModal = () => {
```

Comment:
if only used for the event listener, can you scope this to within the useEffect

Reply:
Updated